docs/dev-tools/zombienet/index.md

```diff
@@ -0,0 +1,600 @@
+
+    !!! note
+        The native provider exclusively utilizes the command config for nodes/collators, which supports both relative and absolute paths. You can employ the `default_command` config to specify the binary for spawning all nodes in the relaychain.
+
+=== "Features"
+    Currently, the Native provider does not execute any additional layers or processes.
+
+## CLI Usage
+
+Zombienet provides a CLI tool that allows interaction with the tool. The CLI can receive commands and flags to perform different kinds of operations. The following sections will guide you through the primary usage of the Zombienet CLI and the available commands and flags.
+
+|  Command  |                                                  Description                                                  |                                                                                                                                                      Arguments                                                                                                                                                       |
+| :-------: | :-----------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|  `spawn`  |                                 Spawn the network defined in the config file                                  |                                                            `<networkConfig>` - a file that declares the desired network to be spawned in `toml` or `json` format. For further information, check out [Configuration Files](#configuration-files) section.                                                            |
+|  `test`   |                                        Run test on the network spawned                                        |                                                                      `<testFile>` - a file that defines assertions and tests against the spawned network, using natural language expressions to evaluate metrics, logs, and built-in functions.                                                                      |
+|  `setup`  |                 Setup is meant for downloading and making dev environment of Zombienet ready.                 |                                                                                         `<binaries>` - executables that will be downloaded and prepared to be used by Zombienet. Options: `polkadot`, `polkadot-parachain`.                                                                                          |
```

Comment:
I would remove the `is meant for...`. Leave it straightforward, like `Set up the dev environment and download requirements` or something like that. Same on Convert